server-spi/src/main/java/org/keycloak/storage/clientscope/ClientScopeStorageProvider.java

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
This should be in server-spi-private, since it is not expected to be supported anytime soon